### ISSUE TITLE
Preview is broken

### ISSUE THREAD

**Issue by drwelby** [DigitalGlobe/gbdxtools#365]
Openlayers library is 404ing.

**Comment by chelm** [comment]
Hmm im not seeing the 404, but am seeing `Cannot read property 'proj' of undefined` but strangely i am not seeing this locally.

**Comment by drwelby** [comment]
I was following the link verbatim, not realizing require.js appends ".js". So the library works. Weirdly enough I changed the link anyway to cdnjs and it started working locally too.

**Comment by chelm** [comment]
that is quite strange

**Comment by chelm** [comment]
Well this is working in dev locally, and in prod GBDXNotebooks we made manual changes that seemed to fix this, though its still unknown exactly why it was broken. Going to close this for now.

**Comment by alopez04** [comment]
Looks like this is still broken. Any chance we can reopen and give this a look?

**Comment by drwelby** [comment]
@alopez04  This was working for me on production just now, can you post the details of the image object you're trying to preview?

**Comment by alopez04** [comment]
@drwelby - Sorry I should have commented back on this... I found that if you DRA when calling CatalogImage, preview() doesn't work.

**Comment by drwelby** [comment]
@alopez04 What do you mean by DRA, and what does 'doesn't work' mean?

`preview()` relies on RDA to provide the map tiles, so anything you do on the Notebook side can't be previewed at the moment.

**Comment by alopez04** [comment]
If "dra=True", the preview returns black slippy map. If "dra=False" then preview returns slippy map with the imagery. Dra is for color adjustments. 

```from gbdxtools.task import env
from gbdxtools import CatalogImage
catalog_id = env.inputs.get('catalog_id', '10400100460FDF00')
bbox = env.inputs.get('bbox', '-104.96481468042478, 39.735437806936154, -104.96083327772796, 39.73819124017263')

image = CatalogImage(catalog_id, band_type="MS", bbox=map(float, bbox.split(",")), dra=True)
image.preview()

**Comment by drwelby** [comment]
OK, I see what's going on. `preview()` adds DRA automatically to the image, so if you create an image with DRA already applied then it confuses RDA.

The workaround for now would be to have two CatalogImage objects, one for preview and one for the other work you are doing. If you only need the preview, then you can skip adding DRA.

**Comment by drwelby** [comment]
Addressed the DRA problem in #621
